typescript/test/deposit-refund.test.ts

```diff
@@ -0,0 +1,170 @@
+import { BigNumber } from "ethers"
+import { MockBitcoinClient } from "./utils/mock-bitcoin-client"
+import bcoin from "bcoin"
+import * as chai from "chai"
+import chaiAsPromised from "chai-as-promised"
+chai.use(chaiAsPromised)
+import { expect } from "chai"
+import { submitDepositRefundTransaction } from "../src/deposit-refund"
+import { TransactionHash, RawTransaction } from "./bitcoin"
+import {
+  refunderPrivateKey,
+  depositRefundOfWitnessDepositAndWitnessRefunderAddress,
+  depositRefundOfNonWitnessDepositAndWitnessRefunderAddress,
+  depositRefundOfWitnessDepositAndNonWitnessRefunderAddress,
+} from "./data/deposit-refund"
+
+describe("Sweep", () => {
```

Comment:
why `Sweep`?

Reply:
Copy-paste error. Fixed in 115e4d6e5d430293d877f5e9a6146a0043449764.